ckanext/aircan/logic/action.py

```
@@ -0,0 +1,179 @@
+                "dag_run_id": dag_run.get("dag_run_id"),
+                "state": "queued",
+                "message": f"Added to the queue to be processed with '{dag_run.get('dag_run_id')}'.",
+                "clear_logs": True,
+            },
+        )
+        return {
+            "dag_run": dag_run,
+            "dag_run_id": dag_run.get("dag_run_id"),
+        }
+    except requests.HTTPError as e:
+        log.error(tk._("Failed to trigger Airflow DAG '%s': %s"), client.dag_id, str(e))
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)